ISSUE_TEMPLATE/membership.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Organization Membership Request
+about: Request membership in a OpenFeature Org
+title: 'REQUEST: New membership for <your-GH-handle>'
+labels: area/github-membership
+assignees: ''
+
+---
+
+<!-- Please remember to change the title of this issue by replacing
+ <your-GH-handle> with the actual GitHub handle -->
+
+### GitHub Username
+
+e.g. (at)example_user
+
+### Requirements
+
+- [ ] I have reviewed the community membership guidelines ADD LINK
+- [ ] I have enabled 2FA on my GitHub account. See https://github.com/settings/security
+- [ ] I have subscribed to the [Slack channel](https://cloud-native.slack.com/archives/C0344AANLA1) (use http://slack.cncf.io/ to get an invite)
+- [ ] I am actively contributing to 1 or more OpenFeature subprojects
+- [ ] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines. Among other requirements, sponsors must be approvers or maintainers of at least one repository in the organization and not both affiliated with the same company
+- [ ] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application
+
+### Sponsors
+
+<!-- Replace (at) with the `@` sign -->
+
+- (at)sponsor-1
+- (at)sponsor-2
+
+Each sponsor should reply to this issue with the comment "*I support*".
+Please remember, it is an applicant's responsibility to get their sponsors' confirmation before submitting the request.
```

Comment:
I don't think verbs use apostrophes `sponsors` is correct I believe

Reply:
here is would be the confirmation of their sponsor